src/code/z_fireobj.c

```diff
@@ -29,21 +29,21 @@ ColliderCylinderInit sFireObjCollisionInit = {
     { 18, 67, 0, { 0, 0, 0 } },
 };
 
-FireObjLightParams sFireObjLightParams[2] = {
-    { 255, 255, 0, 255, 128, 255, 0, 0 },
-    { 0, 200, 128, 128, 0, 127, 127, 0 },
+FireObjColors sFireObjColors[] = {
+    { { 255, 255, 0, 255 }, 128, { 255, 0, 0 } },
 };
```

Comment:
So... this is an array of only one element?

Reply:
Yeah, it seems both `sFireObjColors` and `sFireObjLightParams` are arrays of just one element.